Cookbook/Technical-Documents/Localizations.md

```diff
@@ -0,0 +1,59 @@
+# How to deal with Localizations [draft]
+
+In project we can add a new Localization. But that is not in target level. Babylon needs to have a different sets of languages/locales per product.
+
+Babylon:
+  - British English
+  - Arabic (soon), etc 
+
+NHS111 and Bupa:
+  - British English
+
+Telus:
+  - Canadian English
+  - Canadian French (soon)
+
+Babylon US:
+  - American English
+  - American Spanish
+
+## Build Settings
+### Understanding `app_languages` variable
+Given that, we manage our languages in an array in Build Settings in each target.
+
+| Babylon UK | Babylon US | Telus |
+|--------|--------|--------|
+| `en.lproj/*` | `en-US.lproj/*` `es-US.lproj/*` | `en.lproj/*` |
+
+**I.E we control the languages for each target editing the `APP_LANGUAGES` under _User-Defined_**
```

Comment:
Please add screenshots on this area, for more junior devs to understand what this means.

Reply:
My plan is to add screenshots in all sections. It's an outline (a draft, with a few content).

Reply:
![Screen Shot 2019-07-04 at 15 56 21](https://user-images.githubusercontent.com/17405823/60727612-3b182800-9f36-11e9-9490-1d62e3594d62.png)